### PR TITLE
Fixed namespace block generation

### PR DESCRIPTION
**Pull request by mrtnzlml**
Let's be honest. I am not very sure where is the problem exactly and how to fix it properly. But in my project there was problem with the AOP object cache generation. It was generating class with wrong block namespace (`Parse Error: syntax error, unexpected '{', expecting identifier (T_STRING)`):

``` php
<?php

namespace Kdyby\Aop_CG\ { //<<< HERE

use Kdyby\Aop\Pointcut\Matcher\Criteria;
use Symfony\Component\PropertyAccess\PropertyAccess;

final class Model_PagesClass_83_Model_Pages extends \Model\Pages
{
//...
```

It's probably because there is not system container name in the `$builder->parameters['container']['class']` in new version of Nette (is that correct?). Feel free to implement it yourself. It's just an issue report... (-:

And I hope it's not bug in my code...

### CODE REVIEW

**Comment by fprochazka** [comment]
Nice! I would have probably fixed it very similarly. Would you mind creating a new test case, with the specific code, that broke it for you? Or simply removing the artifically provided classname breaks it?

File: tests/KdybyTests/Aop/Extension.phpt [hunk 1]
```diff
@@ -40,7 +40,6 @@ class ExtensionTest extends Tester\TestCase
 	{
 		$config = new Nette\Configurator();
 		$config->setTempDirectory(TEMP_DIR);
-		$config->addParameters(array('container' => array('class' => 'SystemContainer_' . md5($configFile . microtime()))));
```
Comment (fprochazka) [inline tests/KdybyTests/Aop/Extension.phpt]: This probably has to be solved somehow for 2.2, doesn't it?

Comment (mrtnzlml) [inline tests/KdybyTests/Aop/Extension.phpt]: With 2.2 it's ok. In 2.3 there is a new name convention for system container (like `Container_05082f636f`).

**Comment by mrtnzlml** [comment]
Yes, removing this line is enough for negative test results (with the same error of course)...

**Comment by fprochazka** [comment]
It was required, because there was a bug in nette :) I assume now it's fixed.

**Comment by mrtnzlml** [comment]
Oh, I didn't know that, sorry. Do you have an idea how to create better test (if it's necessary)?

**Comment by fprochazka** [comment]
Well, master branch is currently designed to be used with 2.2 and 2.3, so I don't see a problem with dropping that "fix" (which is hiding a bug).

**Comment by mrtnzlml** [comment]
Ping @fprochazka... Should I somehow improve this PR? It's weird that nobody noticed this issue yet.

**Comment by fprochazka** [comment]
Thank you!

**Comment by fprochazka** [comment]
Oh wow, there is actually no release yet!

**Pull request closed without merging**